src/css/UnderHeader.scss

```diff
@@ -30,133 +26,87 @@ body {
       font-style: normal;
       font-weight: 300;
       font-size: 18px;
+      width: 100%;
+      margin: 0px;
       padding: 10px 20px;
       background: #FFFFFF;
       border: 1px solid rgba(0, 0, 0, 0.2);
       box-sizing: border-box;
       border-radius: 4px;
     }
-  }
-  
-}
-.bodyWrap {
-  background-color: rgb(206,232,248);
-  height: calc(100vh - 117px);
-  box-shadow: inset 0px 20px 5px -14px rgb(0 0 0 / 32%);
-  width: 100%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-
-  .body {
-    width: 1000px;
-    display: flex;
-    align-items: center;
-    justify-content: space-around;
-
-    svg {
-
-    }
-
-    
-  }
-
-  form {
-    background-color: #fff;
-    border-radius: 4px;
-    padding: 0 40px 25px 40px;
-    width: 350px;
-    height: auto;
-    display: flex;
-
-    flex-direction: column;
-
-    p {
-      font-size: 15px;
-      color: rgb(255, 86, 86);
-      animation-duration: 0.1s;
-      animation-name: errPopUp;
-
-    }
-
-    h1 {
-      font-family: 'Roboto', sans-serif;
-      font-style: normal;
-      font-weight: 300;
-      font-size: 36px;
-      line-height: 42px;
-      margin: 40px 0 40px 40px;
-    }
-
-    label {
+    input[type=date] {
       font-family: 'Roboto', sans-serif;
       font-style: normal;
       font-weight: 300;
       font-size: 18px;
-      margin: 8px;
-      line-height: 21px;
+      width: 100%;
+      margin: 0px;
+      padding: 8px 20px ;
+      background: #FFFFFF;
+      border: 1px solid rgba(0, 0, 0, 0.2);
+      box-sizing: border-box;
+      border-radius: 4px;
     }
-     input {
+    select {
       font-family: 'Roboto', sans-serif;
       font-style: normal;
       font-weight: 300;
       font-size: 18px;
-      padding: 10px 20px;
+      width: 100%;
+      margin: 0px;
+      padding: 9px 20px;
       background: #FFFFFF;
       border: 1px solid rgba(0, 0, 0, 0.2);
       box-sizing: border-box;
       border-radius: 4px;
-     }
-     
-     button {
+    }
+    button {
       background: #FFFFFF;
       border: 1px solid rgba(0, 0, 0, 0.2);
       box-sizing: border-box;
       border-radius: 4px;
+      align-self: flex-end;
+      margin: 0px 0 5px 5px;
       padding: 10px;
       font-family: 'Roboto', sans-serif;
       width: fit-content;
       font-style: normal;
       font-weight: 300;
       font-size: 18px;
-      align-self: flex-end;
-      margin: 20px 0 0 0;
+    
+      margin-left: 15px;
       line-height: 21px;
       transition: 0.1s;
      }
     
      button:hover {
       background: #C5E9FF;
-      border: 1px solid #C5E9FF;
+
+      border: 1px solid rgba(0, 0, 0, 0.2);
       transition: 0.1s;
       color: #fff;
      }
-
-     a {
-      align-self: flex-end;
-      margin: 8px;
+     
+     label {
+      margin: 8px 0;
+      font-family: 'Roboto';
+      font-style: normal;
+      font-weight: 300;
+      font-size: 18px;
+      line-height: 21px;
      }
   }
```

Comment:
please check all the styles for the design. And nowhere should be the exact values